MyArrayList/Program.cs

```diff
@@ -164,11 +181,13 @@ public void GetIndexByValue(int Value)
         }
 
     }
+    
     // returns the length of of list
     public int Length()
```

Comment:
This can be done as Property. so it will be more convenient to use it.
old-school style
```
    public int Length 
    {
        get 
        {
            return _list.Length;
        }
    }
```
or some modern style
`public int Length => _list.Length;`